gitlab/global/stages/40-delivery/release.yaml

```diff
@@ -6,8 +6,8 @@ release:
   script:
     - |
       set -eux
-      if echo "$CI_COMMIT_MESSAGE" | grep -Eq "^Merge (remote-tracking )?branch '(origin\/)?chore/bump-([0-9]+\.[0-9]+\.[0-9]+)'$"; then
-          VERSION=$(echo "$CI_COMMIT_MESSAGE" | sed -n -E "s/^Merge (remote-tracking )?branch '(origin\/)?chore\/bump-[^']*)'( into '[^']*')?$/\3/p")
+      if echo "$CI_COMMIT_MESSAGE" | grep -Eq "^Merge (remote-tracking|remote|tracking)? ?branch '(origin\/)?chore\/bump-([0-9]+\.[0-9]+\.[0-9]+)' into '\w+'$"; then
+          VERSION=$(echo "$CI_COMMIT_MESSAGE" | sed -n -E "s/^Merge.*branch '(origin\/)?chore\/bump-([^']+)' into '.*'$/\2/p")
```

Comment:
Very cool my friend. Could you please check here: https://github.com/rios0rios0/pipelines/blob/main/azure-devops/global/stages/40-delivery/release.yaml as well?